portfolio/src/main/webapp/script.js

```diff
@@ -45,4 +30,22 @@ function popupDetail(id){
         detail.style.visibility = "hidden";
         button.innerHTML = "Show Details";
     }
+}
+
+function createListElement(text) {
+    const liElement = document.createElement('li');
+    liElement.innerText = text;
+    return liElement;
+}
+
+async function loadComments(){
+    const response = await fetch("/comments");
+    var comments = await response.text();
+    comments = JSON.parse(comments);
+
+    const commentsListElement = document.getElementById('comments');
+    commentsListElement.innerHTML = '';
+    for(var i=0;i<comments.length;i++){
```

Comment:
Please add whitespaces after ; and around +, = etc operators
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

Reply:
I think, I added whitespaces in a proper way in my last commit.